api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -52,6 +52,7 @@
 public class ConceptServiceImplTest extends BaseContextSensitiveTest {
 	
 	protected ConceptService conceptService = null;
+	protected String originalText = "OriginalText";
```

Comment:
Because this is duplicated in only one method, then you do not need to declare it at class level.